src/test/crimson/gtest_seastar.cc

```diff
@@ -15,21 +15,29 @@ SeastarRunner seastar_test_suite_t::seastar_env;
 
 int main(int argc, char **argv)
 {
-  ::testing::InitGoogleTest(&argc, argv);
-
-  seastar_test_suite_t::seastar_env.init(argc, argv);
-
-  // HACK: differntiate between the `make check` bot and human user
+  // preprocess args
+  std::vector<const char*> args;
+  bool global_log_level_is_set = false;
+  for (int i = 0; i < argc; ++i) {
+    if (std::strncmp(*argv, "--default-log-level", 19) == 0) {
```

Comment:
Nobody likes to count. Maybe let's do the following instead?

```suggestion
    if (std::strncmp(*argv, "--default-log-level", std::strlen("--default-log-level")) == 0) {
```

Reply:
and nobody likes to repeat =) why not 
```c++
bool global_log_level_is_set = std::any_of(argv, argv + argc, [=](const char* arg) {
  using namespace std::literals;
  return "--default-log-level"sv.compare(*argv) == 0;
});
```

Reply:
Fixed, I still used `std::strncmp()` because I also want `unittest-seastore-cache --default-log-level=info` to work.